code/modules/mob/living/carbon/species.dm

```diff
@@ -337,7 +337,7 @@ var/global/list/whitelisted_species = list("Human")
 		"brain" =    /datum/organ/internal/brain,
 		)
 
-	move_speed_multiplier = 1.5
+	move_speed_multiplier = 1
```

Comment:
You can actually just remove this line altogether. The default speed of `/datum/species` is 1, so all you have to do is not override it.